api/pkg/transformer/feast/storage.go

```diff
@@ -6,24 +6,23 @@ import (
 	"errors"
 	"fmt"
 	"sort"
-	"strings"
 	"time"
 
-	"github.com/feast-dev/feast/sdk/go"
-	"github.com/feast-dev/feast/sdk/go/protos/feast/core"
+	feast "github.com/feast-dev/feast/sdk/go"
 	"github.com/feast-dev/feast/sdk/go/protos/feast/serving"
 	"github.com/feast-dev/feast/sdk/go/protos/feast/storage"
 	"github.com/feast-dev/feast/sdk/go/protos/feast/types"
 	"github.com/go-redis/redis/v8"
+	"github.com/gojek/merlin/pkg/transformer/spec"
 	"github.com/golang/protobuf/proto"
 	"github.com/spaolacci/murmur3"
 	"google.golang.org/protobuf/types/known/durationpb"
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-func NewDirectStorageClient(storage *OnlineStorage, featureTables []*core.FeatureTableSpec) (StorageClient, error) {
+func NewDirectStorageClient(storage *spec.OnlineStorage, featureTablesMetadata []*spec.FeatureTableMetadata) (StorageClient, error) {
 	switch storage.Storage.(type) {
-	case *OnlineStorage_Redis:
+	case *spec.OnlineStorage_Redis:
```

Comment:
We have only 2 types of `ServingSource`, how do we differentiate between redis and redis cluster? @tiopramayudi @khorshuheng

Reply:
it depends on how orchestrator(merlin api) setup feast config. In standard transformer merlin will choose one of redis or redis cluster for REDIS source to pass to standard transformer.

Reply:
in this PR I only consider redis cluster, but I'll update it to acommodate user to use single redis storage